### DISCUSSION TITLE
Tab navigation cmd-shift-§

### DISCUSSION THREAD

**Discussion by peterpeterparker** [zed-industries/zed#12228] [Feature Requests]
Hi 👋,

I can set up a custom key binding for `cmd-§` but, I am unable to set up one for `cmd-shift-§`. I'm not sure if it's expected, therefore wondering if I am missing something?

```
[
  {
    "context": "Editor",
    "bindings": {
      "cmd-§": "pane::ActivateNextItem", // <------- Works
      "cmd-shift-§": "pane::ActivatePrevItem" // <----- Does not work, regardless of the shortcut
    }
  }
]
```

I've got a Swiss-french keyboard on MacOS with Jetbrains set as keymap.

**Comment by SomeoneToIgnore** [comment]
Hey, the culprit here is how Zed currently treats keybindings.
Here's a good example: https://github.com/zed-industries/zed/blob/8ed861f5e9605ea45a50ab2154d73b1813e33b95/assets/keymaps/default-macos.json#L536

On US keyboards, `ctrl-:` is input as `ctrl-shift-;`, but using the latter would not work: Zed tracks the "text" you input with the keypresses.

So, whatever's `shift-§` producing in the text editor (is that `%`?) has to be used instead.

**Reply by peterpeterparker** [reply]
Ah gotcha, I'm dumb! I thought about that and transleted `cmd-shift-§` to `cmd-shift-°` without removing the `shift-`. `cmd-°` works out, makes sense. Thanks for the feedback.
